core/src/main/kotlin/ch/ergon/dope/validtype/ValidType.kt

```diff
@@ -1,6 +1,6 @@
 package ch.ergon.dope.validtype
 
-interface ValidType
+sealed interface ValidType
```

Comment:
Why is it sealed?

Reply:
On request of @m0ne, I changed it to sealed.

Reply:
Why shouldn't it be sealed?